cli/containerd-shim-kata-v2/main.go

```diff
@@ -1,13 +1,16 @@
-
+// Copyright (c) 2018 HyperHQ Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
Please rework the commit:

    containerd-shim-kata: Support shim's start subcommand
    
    When creating a container in an existing sandbox,
    this shim should return its sock address and exit
    directly.

There is some rework of a bunch of structures that are not related to the support of the `start` subcommand.